src/main/java/com/zollingpaper/backend/board/dto/BoardSaveReponse.java

```diff
@@ -0,0 +1,4 @@
+package com.zollingpaper.backend.board.dto;
+
+public record BoardSaveReponse() {
```

Comment:
```suggestion
public record BoardSaveResponse() {
```

오타 발견! 🧐
이 레코드는 추후 사용을 위해 미리 만들어 놓은 것일까요?

Reply:
오타가 있었군요 👀 잡아주셔서 감사합니다.
며칠 전에 말씀하셨던 내용을 적용해보고자 만들어봤습니다.
생성 api 의 반환값을 void로 할 것인지, 생성에 성공한 값을 반환할 것인지 고민을 잠깐 해보았는데요, 어떤 방안이 더 좋을까요?

Reply:
저는 생성에 성공한 값을 반환하는 걸 선호하는 편이긴한데 어떤 방식이든 괜찮습니다!